### ISSUE TITLE
32bit avisynth+ rife version?

### ISSUE THREAD

**Issue by Usulyre** [Asd-g/AviSynthPlus-RIFE#9]
Is it possible to have a version for 32 bit avisynth+ ?

Just wondering.

**Comment by Asd-g** [comment]
Yes, its possible only if you really need it and it probably will be only one time (no regular 32-bit builds).

**Comment by Asd-g** [comment]
Reopen if needed.
